tests/test_coordinates_new.py

```diff
@@ -0,0 +1,562 @@
+import numpy as np
```

Comment:
Maybe rename the old _test_coordinates.py_ file instead, so it can be removed with the release of 0.8.0?

Reply:
i tried it ealier, but then the diffs are not tracked properly